src/errors/error.es6.js

```diff
@@ -0,0 +1,15 @@
+export default class ErrorClass {
```

Comment:
Making FakeError act as close as possible to the native Error implementation would be nice; but that's mostly what I've built anyways, grabbing stack traces and messages. Only reason not to is that browserify would have to pull in `utils`. I'll ship as-is, and if we find that something's lacking we can revisit.